### ISSUE TITLE
It would be good for debugging to be able to request a specific clip from the backend

### ISSUE THREAD

**Issue by ftyers** [omnilingo/omnilingo#109]
e.g. if you know that there is some bug in a clip, it would be useful to be able to reproduce it.

**Comment by d33tah** [comment]
Wouldn't we have to make the code more deterministic overall in order to achieve this? Make the project generate same data on different computers?

**Comment by ftyers** [comment]
No, you would just add a `&clipId=` thing to `get_clips`, all we'd need is an index keyed on that.
